spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET /users' do
+    before(:example) { get '/users' }
+    it 'should check the responce is comming' do
+      expect(response).to have_http_status(:ok)
+    end
+    it('is correct template rendered') do
+      expect(response).to render_template(:index)
+    end
+    it 'should check the responce is comming' do
+      expect(response.body.size).to eq(6635)
+    end
```

Comment:
- [ ] You did a great job fixing almost all the tests which were failing, but one test needs your attention. I have attached an image for your reference.

![test](https://user-images.githubusercontent.com/53124548/230115854-078db97e-b3f2-4361-973a-59af5f89b907.png)